src/platform/ios.rs

```diff
@@ -1,16 +1,25 @@
 //! # iOS / UIKit
 //!
-//! Winit has an OS requirement of iOS 8 or higher, and is regularly tested on
-//! iOS 9.3.
+//! Winit has [the same iOS version requirements as `rustc`][rustc-ios-version], although it's
+//! frequently only tested on newer iOS versions.
 //!
-//! ## Window initialization
+//! [rustc-ios-version]: https://doc.rust-lang.org/stable/rustc/platform-support/apple-ios.html#os-version
 //!
-//! iOS's main `UIApplicationMain` does some init work that's required by all
-//! UI-related code (see issue [#1705]). It is best to create your windows
-//! inside [`ApplicationHandler::resumed`].
+//! ## Running on Mac Catalyst
 //!
-//! [#1705]: https://github.com/rust-windowing/winit/issues/1705
-//! [`ApplicationHandler::resumed`]: crate::application::ApplicationHandler::resumed
+//! Mac Catalyst allows running applications using UIKit on macOS, which can be very useful for
+//! testing. Winit somewhat supports this, though you'll need to bundle your application before
```

Comment:
Should we elaborate on "somewhat"?

Reply:
I originally wrote "somewhat" because click events aren't working on Mac Catalyst, but it seems like [they aren't elsewhere either](https://github.com/rust-windowing/winit/issues/2962), so it's not a problem tied to Mac Catalyst, it's a more general problem

Reply:
Uff.
Personally I would either drop "somewhat" or elaborate, but yeah, hairy situation.

Reply:
I've removed "somewhat" now, thanks for the input - will request it again once I'm done writing